spec/lib/authorities/linked_data/find_term_spec.rb

```diff
@@ -90,90 +90,131 @@
             .to include('Cornell University', 'Ithaca (N.Y.). Cornell University', "Kornel\\xCA\\xB9skii universitet",
                         "K\\xCA\\xBBang-nai-erh ta hs\\xC3\\xBCeh")
         end
+
+        context "ID in graph doesn't match ID in request URI" do
+          before do
+            stub_request(:get, 'http://id.worldcat.org/fast/530369')
+              .to_return(status: 200, body: webmock_fixture('lod_oclc_term_bad_id.nt'), headers: { 'Content-Type' => 'application/ntriples' })
+          end
+
+          it 'raises DataNormalizationError' do
+            expect { lod_oclc.find('530369') }.to raise_error Qa::DataNormalizationError, "Unable to extract URI based on ID: 530369"
+          end
+        end
       end
     end
 
     context 'in LOC authority' do
       context 'term found' do
-        before do
-          stub_request(:get, 'http://id.loc.gov/authorities/subjects/sh85118553')
-            .to_return(status: 200, body: webmock_fixture('lod_loc_term_found.rdf.xml'), headers: { 'Content-Type' => 'application/rdf+xml' })
-          stub_request(:get, 'http://id.loc.gov/authorities/subjects/sh1234')
-            .to_return(status: 200, body: webmock_fixture('lod_loc_second_term_found.rdf.xml'), headers: { 'Content-Type' => 'application/rdf+xml' })
-        end
+        context 'when id requires special processing for <blank> in id' do
```

Comment:
This is mostly a refactor moving the stubs closer to the tests that use them for readability purposes only.